### DISCUSSION TITLE
Issue with deployment run, "sharp" moduleCannot find

### DISCUSSION THREAD

**Discussion by ivsaneesh** [vendurehq/vendure#1704] [Q&A]
Created build using `npm run build`
When trying to run the application getting the error from server

`﻿Something went wrong installing the "sharp" moduleCannot find module '../build/Release/sharp-linux-x64.node'
﻿Require stack:
﻿- /home/68460.cloudwaysapps.com/hx/public_html/node_modules/sharp/lib/sharp.js
﻿- /home/68460.cloudwaysapps.com/hx/public_html/node_modules/sharp/lib/constructor.js
﻿- /home/68460.cloudwaysapps.com/hx/public_html/node_modules/sharp/lib/index.js
﻿- /home/68460.cloudwaysapps.com/hx/public_html/node_modules/@vendure/asset-server-plugin/lib/src/sharp-asset-preview-strategy.js
﻿- /home/68460.cloudwaysapps.com/hx/public_html/node_modules/@vendure/asset-server-plugin/lib/src/plugin.js
﻿- /home/68460.cloudwaysapps.com/hx/public_html/node_modules/@vendure/asset-server-plugin/lib/index.js
﻿- /home/68460.cloudwaysapps.com/hx/public_html/dist/vendure-config.js
﻿- /home/68460.cloudwaysapps.com/hx/public_html/dist/index.js`

In the composer.log file there are some dependencies in which the sharp module is not working maybe it's conflicting. 
I think composer.log file metadata should be defined package.json composer will read and install it.

Any idea on this.
Please replay

**Comment by michaelbromley** [comment]
Hi, what node version are you running on the server?

**Reply by ivsaneesh** [reply]
In server node version is v16.13.2
and in local it is v16.14.0

**Comment by denerFernandes** [comment]
Are you installing locally? docker? Make sure sharp is installing correctly, try installing it in some other project, something common is the lack of libvips, make sure it is installed correctly.

**Reply by ivsaneesh** [reply]
Not using docker.

**Comment by ivsaneesh** [comment]
Now getting new error. node_modules/bcrypt/lib/binding/napi-v3/bcrypt_lib.node: invalid ELF header

**Reply by ivsaneesh** [reply]
Also getting admin-api?languageCode=en: 503 Service Unavailable
